db/data/20210514141445_change_gcrf_strategic_area_code_2_to_3.rb

```diff
@@ -0,0 +1,6 @@
+# Run me with `rails runner db/data/20210514141445_change_gcrf_strategic_area_code_2_to_3.rb`
+
+Activity.where("gcrf_strategic_area @> '{2}'::integer[]").each do |activity|
+  gcrf_strategic_areas = activity.gcrf_strategic_area.map { |i| i == 2 ? 3 : i }
```

Comment:
If multiple choice is an option, do we need to remove any potential duplicate `3`s at the end? I don't suppose the database automagically enforces a set with distinct values.

Reply:
I don't think it's too much of a concern. All the existing values have only one value anyway, and we're going to be migrating this to only support one value soon anyway.

Reply:
OK, then all it needs is rebasing.